### ISSUE TITLE
Add easy access links to the different APIs

### ISSUE THREAD

**Issue by pierremoreau** [MightyCreak/mesamatrix#189]
At the moment there are no links to the different sections, so accessing Vulkan or OpenCL requires scrolling (almost) all the way down.

It would be great to have a list of the different APIs at the top, and clicking on the API name would take you to the appropriate section. Additionally (and probably collapsed by default to avoid taking too much space), after each API there could be a list of the different versions available and clicking on a version would take you to the corresponding sub-section; links could also be found there to take you to the different extension sections.

It could look something like the following by default:

* [OpenGL]()
  \>
* [OpenGL ES]()
  \>
* [Vulkan]()
  \>
* [OpenCL]()
  \>

And if expanding the Vulkan section:

* [OpenGL]()
  \>
* [OpenGL ES]()
  \>
* [Vulkan]()
  v
  * [Vulkan 1.2]()
  * [Vulkan 1.1]()
  * [Vulkan 1.0]()
  * [Extensions that are not part of any Vulkan version]()
* [OpenCL]()
  \>

**Comment by MightyCreak** [comment]
You're right. In the old times, Mesamatrix was much simpler, but it grew and now there are a lot of information to show.

I'm thinking of something like you said to ease navigation. I'd like to simply collapse the extensions sections by default, only showing the leaderboards for each section.

This should reduce the overall vertical space. But as always with frontend dev, it requires time to test what's best.

**Comment by pierremoreau** [comment]
Awesome, thank you!

Mmh I see, the OpenGL/ OpenGL ES situation is a bit tricky regarding the extensions.
